code/datums/components/chem_booster.dm

```diff
@@ -0,0 +1,237 @@
+/**
+ *	Chem booster component
+ *
+ *	This component stores "green blood" and uses it to increase effect_str of chems in the user's body.
+ *
+ */
+/datum/component/chem_booster
+	var/mob/living/carbon/human/wearer
+
```

Comment:
Is there a particular reason you use hard refs instead of signal args to get a wearer reference here?